src/Symfony/Component/Notifier/Bridge/Slack/SlackTransport.php

```diff
@@ -91,6 +91,9 @@ protected function doSend(MessageInterface $message): SentMessage
             throw new TransportException(sprintf('Unable to post the Slack message: "%s".', $result['error']), $response);
         }
 
-        return new SentMessage($message, (string) $this);
+        $sentMessage = new SentMessage($message, (string) $this);
+        $sentMessage->setMessageId($result['ts']);
```

Comment:
Shall we check if the key exists in the array?

Reply:
We already check for "ok" status. I had looked to another transports/bridges, they did not have even this checks.

Which behavior do you expect, if by some unknown reason slack return response with ok status, but with no messageid specified?

Reply:
Yes, but ok let's keep it